python/GafferOSL/FocalBlur.gfr

```diff
@@ -0,0 +1,889 @@
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthPlaneScalingFactor"], 'description', 'The ratio between pixel radius for each depth plane. This controls how many depth planes are needed, and how quickly occlusion is applied as depth increases. Lower values are more efficient, and capture less occulsion, higher values result in more occlusion. The default value of 0.5 is good for both performance and aesthetics ( setting it too high can result in too much occlusion between features that are actually close together ).' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthPlaneScalingFactor"], 'layout:section', 'Advanced' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["depthPlaneScalingFactor"], 'layout:index', 21 )
+__children["FocalBlur"]["Merge26"]["enabled"].setInput( __children["FocalBlur"]["infilling"] )
+__children["FocalBlur"]["Merge26"]["in"][0].setInput( __children["FocalBlur"]["DiskBlur29"]["out"] )
+__children["FocalBlur"]["Merge26"]["in"][1].setInput( __children["FocalBlur"]["Premultiply"]["out"] )
+__children["FocalBlur"]["Merge26"]["__uiPosition"].setValue( imath.V2f( 825.123169, -318.060547 ) )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["approximationThreshold"], 'nodule:type', '' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["approximationThreshold"], 'description', 'The maximum acceptable error caused by omitting anti-aliasing for a particular disk in the internal disk blur. Since very large disks often contribute very little to each individual output pixel, omitting anti-aliasing for them can provide a substantial speed improvement.' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["approximationThreshold"], 'layout:section', 'Advanced' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["approximationThreshold"], 'layout:index', 22 )
+__children["FocalBlur"]["Dot15"]["in"].setInput( __children["FocalBlur"]["BoxIn"]["out"] )
+__children["FocalBlur"]["Dot15"]["__uiPosition"].setValue( imath.V2f( 677.8974, 101.000412 ) )
+__children["FocalBlur"]["Dot16"]["in"].setInput( __children["FocalBlur"]["Dot15"]["out"] )
+__children["FocalBlur"]["Dot16"]["__uiPosition"].setValue( imath.V2f( 677.8974, -350.336182 ) )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["backgroundZValue"], 'layout:section', 'Depth' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["backgroundZValue"], 'layout:index', 20 )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["backgroundZValue"], 'nodule:type', '' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["backgroundZValue"], 'layout:visibilityActivator', 'depthInterpretationFilteredWithBackground' )
+Gaffer.Metadata.registerValue( __children["FocalBlur"]["backgroundZValue"], 'description', 'Set this if your renderer writes a constant Z value in empty pixels that we need to subtract off before we can upremultiply the edges. In Arnold this comes from the far clip plane of the camera you rendered with.' )
```

Comment:
Maybe this description would be more clear along these lines?

The value to be subtracted before the depth channel is unpremultiplied. When rendering filtered depth from Arnold, this value is the far clip plane of the render camera.

Reply:
Hmm, I still think that mentioning how you would identify an image where this is needed is the first thing a user needs to know about this value. I've shifted to using shorter sentences more in line with your wording, though:

`Set this if your renderer writes a constant Z value in empty pixels. This value will be subtracted before the depth channel is unpremultiplied. When rendering filtered depth from Arnold, this value is the far clip plane of the render camera.`

Reply:
Oh, I see you've moved that part into the description for depthInterpretation, sure, that makes sense, I'll just go with your wording.